src/controller/python/ChipDeviceController-StorageDelegate.cpp

```diff
@@ -56,28 +51,28 @@ CHIP_ERROR PythonPersistentStorageDelegate::SyncGetKeyValue(const char * key, ch
 
     if (size < neededSize)
```

Comment:
Should neededSize still have the `+1` in its computation above if we're making this change to binary?

Reply:
Removed the +1 - when having binary data we should not need a null terminator.